AmplifyTestCommon/Models/NonModel/DynamicModel.swift

```diff
@@ -0,0 +1,68 @@
+//
+// Copyright 2018-2020 Amazon.com,
+// Inc. or its affiliates. All Rights Reserved.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+
+import Amplify
+
+struct DynamicModel: Model, JSONValueHolder {
```

Comment:
is this something that should live in Amplify core or AWSPluginsCore instead of AmplifyTestCommon?